terminus/tests/custom_assertions_mixin.py

```diff
@@ -46,8 +46,8 @@ def assertArcAlmostEqual(self, arc1, arc2, places=7, msg=None):
                                     arc2.start_point(),
                                     places=places,
                                     msg=start_point_msg)
-        self.assertAlmostEqual(arc1.theta(),
-                               arc2.theta(),
+        self.assertAlmostEqual(arc1.theta() % 360,
```

Comment:
Do we really need this? I would so much prefer our angles to be already normalized here